lib/fs_utils/file_list.js

```diff
@@ -163,6 +164,7 @@ class FileList extends EventEmitter {
   copy(asset) {
     const path = asset.path;
     const resetCopy = (p) => {
+      if (this.disposed) { return p; }
```

Comment:
I think there are situations where the FileList is disposed, but some file operations are still running. So when their callbacks are resolved, some properties are null, or read-only because we froze them in the dispose method. Accessing these properties give errors. I added this disposed flag to skip some operations when the object was already disposed.